.env.example

```diff
@@ -16,3 +16,14 @@ GC_NOTIFY_CALLBACK_BEARER_TOKEN=
 
 # Used when running against LocalStack
 LOCAL_AWS_ENDPOINT=http://localhost:4566
+
+# Local dev 
+# Temporary token templateID
+TEMPORAY_TOKEN_TEMPLATE_ID=b6885d06-d10a-422a-973f-05e274d9aa86
+
+# A link to GC Nofify service like staging or production
+GC_NOTIFY_URL=https://notification.canada.ca
```

Comment:
This is the incorrect url. It should be https://api.notification.canada.ca . We should probably go back to hardcoding this